specs/version_format_specification.md

```diff
@@ -63,6 +63,22 @@ strings are found to compare as different. In a loop:
    of the whole string), if the other substring has remaining letters, it compares higher.
    Otherwise, the comparison continues at the following characters at point 1.
 
+## Comparison With Other Specifications
+Other specifications exist to mandate version formats:
+
+- [RPM Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/)
+- [Debian Policy](https://www.debian.org/doc/debian-policy/ch-controlfields.html#version)
+- [Semantic Versioning](https://semver.org/)
+
+All of these, including the present document, share some commonalities but are also
+incompatible in some ways, as they all evolved in different environments. The main
+differences are as follows.
+
+- to separate components RPM uses `-`, DEB uses `_` and SemVer does not specify anything
```

Comment:
Hmm, semver only talks about the version part of the string… So by definition it doesn't care about how the version is separated from other components in a file name or such.

Also, RPM uses `-` with some positional logic. (I think the last two dashes separate the release and version, but I'd need to check that.)

Reply:
updated